include/aws/common/mutex.h

```diff
@@ -19,7 +19,7 @@
 #include <aws/common/common.h>
 #ifdef _WIN32
 /* NOTE: Do not use this macro before including Windows.h */
-#    define AWSMUTEX_TO_WINDOWS(pCV) (PSRWLOCK) pCV
+#    define AWSMUTEX_TO_WINDOWS(pCV) AWS_CONTAINER_OF((pCV), PSRWLOCK, mutex_handle)
```

Comment:
```AWS_MUTEX_TO_WINDOWS``` if this is public?

Reply:
It's public because `windows/condition_variable.c` is using it